uui/components/tables/DataRowsContainer/DataRowsGroups.tsx

```diff
@@ -23,10 +23,10 @@ function DataRowsGroup<TItem, TId>({
     top = 1,
 }: DataRowsGroupProps<TItem, TId>) {
     const rowRef = useRef<HTMLDivElement>();
-    const childrenPinnedTop = row.isPinned ? (top + (rowRef.current?.clientHeight ?? 0) - 0.5) : top;
+    const childrenPinnedTop = row.isPinned ? (top + (rowRef.current?.clientHeight ?? 0) - 1) : top;
```

Comment:
Let's add comment why we subtract 1 here

Reply:
Done,.